helm/credentiald-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: credentiald
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
```

Comment:
I think we only need `create` currently, as we're not getting or deleting secrets

Reply:
done